apps/server/src/shared/testing/factory/user-and-account.test.factory.ts

```diff
@@ -0,0 +1,73 @@
+import { Permission, Role, School, EntityId, Account, User, BaseEntity } from '@shared/domain';
+import { ObjectId } from 'bson';
+import _ from 'lodash';
+import { userPermissions, studentPermissions, teacherPermissions, adminPermissions } from '../user-role-permissions';
+import { accountFactory } from './account.factory';
+import { roleFactory } from './role.factory';
+import { schoolFactory } from './school.factory';
+import { userFactory } from './user.factory';
+
+export interface UserAndAccountParams {
+	username?: string;
+	systemId?: EntityId | ObjectId;
+	firstName?: string;
+	lastName?: string;
+	email?: string;
+	school?: School;
+}
+
+export class UserAndAccountTestFactory {
+	private static checkIdExists(entity: BaseEntity): void | Error {
```

Comment:
Der return-type müsste nur void sein. Die Methode returned ja keinen Error.

Reply:
Da ist mir die best pratics in javascript/node nicht geläufig.
In java würde man für check Methoden die Exception mit angeben. Daher vermutlich anstatt Error eher den spezifischen Exception Type. Bisher haben wir das dann für check hier in TS auch so gehandhabt.

Reply:
put it to arc weekly